include/SW_datastructs.h

```diff
@@ -739,7 +743,8 @@ typedef struct {
 } LOG_INFO;
 
 typedef struct {
-	char *InFiles[SW_NFILES];
+	char *InFiles_csv[SW_NFILES];
```

Comment:
This was meant to parallel the naming of _nc variables. But I see your point, this is not meant to hold anything csv-related. I will change that back.

I was planning on splitting the variable into the two when we encounter an output file array for netCDFs so it can happen at the same time we add `OutFiles_nc`.

Reply:
ok makes sense